File: NeuroHand/modules/hand-tracker/ios/HandTrackerModule.swift
import ExpoModulesCore

public class HandTrackerModule: Module {
  public func definition() -> ModuleDefinition {
    Name("HandTracker")

    View(HandTrackerView.self) {
      Events("onHandLandmarks")

      AsyncFunction("takeSnapshot") { (view: HandTrackerView) -> String in
        try await view.takeSnapshot()
      }
    }
  }
}

File: NeuroHand/modules/hand-tracker/android/src/main/java/expo/modules/handtracker/HandTrackerModule.kt
package expo.modules.handtracker

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class HandTrackerModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("HandTracker")

    View(HandTrackerView::class) {
      Events("onHandLandmarks")

      OnViewDestroys { view ->
        view.destroy()
      }
    }
  }
}
